src/builder/BridgeRoutes.sol

```diff
@@ -0,0 +1,465 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.23;
+
+import {IQuarkWallet} from "quark-core/src/interfaces/IQuarkWallet.sol";
+import {TransferActions} from "../DeFiScripts.sol";
+
+contract QuarkBuilder {
+    /* ===== Constants ===== */
+    string constant VERSION = "1.0.0";
+
+    string constant PAYMENT_METHOD_OFFCHAIN = "OFFCHAIN";
+    string constant PAYMENT_METHOD_PAYCALL = "PAY_CALL";
+    string constant PAYMENT_METHOD_QUOTECALL = "QUOTE_CALL";
+
+    string constant ACTION_TYPE_BRIDGE = "BRIDGE";
+    string constant ACTION_TYPE_TRANSFER = "TRANSFER";
+
+    string constant PAYMENT_CURRENCY_USD = "usd";
```

Comment:
Probably no need for payment currencies that are not tokens.

Reply:
Yeah. These constants are currently unused, so will probably be cleaned up in the PR that constructs Quark Operations and action contexts.